test/js/concat.spec.js

```diff
@@ -137,3 +137,59 @@ describe('Plus operator', function () {
     assert.deepEqual(expected, TaintedUtils.getRanges(id, ret))
   })
 })
+
+describe('concat method', () => {
```

Comment:
Please, use the way that we are using in other PRs to test tainted objects, is clearer and easier to implement.
For example, trim:
https://github.com/DataDog/dd-native-iast-taint-tracking-js/blob/main/test/js/trim.spec.js